pkg/serverless/trace/testdata/event_samples/sqs-batch.json

```diff
@@ -36,7 +36,7 @@
             },
             "messageAttributes": {
                 "_datadog": {
-                    "stringValue": "{\"x-datadog-trace-id\":\"2684756524522091840\",\"x-datadog-parent-id\":\"7431398482019833808\",\"x-datadog-sampled\":\"1\",\"x-datadog-sampling-priority\":\"1\"}",
+                    "stringValue": "{\"x-datadog-trace-id\":\"2684756524522091841\",\"x-datadog-parent-id\":\"7431398482019833809\",\"x-datadog-sampled\":\"1\",\"x-datadog-sampling-priority\":\"1\"}",
```

Comment:
This sample event has two `SQSMessage`s in it. I updated the second one to use a different trace and parent id. This ensures in our testing that we're pulling from the first message only.